Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM python:3.12-slim
 LABEL maintainer=tech@civisanalytics.com
 
-ENV APP_CONFIG_FILE=/app/civis_app/config/production.py
+ENV APP_CONFIG_FILE=config/production.py
+ENV PIP_ROOT_USER_ACTION=ignore
```

Comment:
Using the root user can sometimes be sketchy, but it seems like this is accepted practice in this situation, and that a bad actor couldn't do damage from this.